ste/xfer-anyToRemote-folder.go

```diff
@@ -82,7 +82,7 @@ func anyToRemote_folder(jptm IJobPartTransferMgr, info *TransferInfo, pacer pace
 	} else {
 
 		t := jptm.GetFolderCreationTracker()
-		defer t.StopTracking(s.DirUrlToString()) // don't need it after this routine
+		//defer t.StopTracking(s.DirUrlToString()) // don't need it after this routine
```

Comment:
@adreed-msft We see this map deletion was introduced to save some memory footprints, but this is causing duplicate directory creation calls as the directory creation relies on entry being present in the map. Kindly verify.

Reply:
I like the idea of a trie for mapping